config/a8c-for-agencies-stage.json

```diff
@@ -0,0 +1,22 @@
+{
+	"env": "production",
+	"env_id": "a8c-for-agencies-stage",
+	"favicon_url": "/calypso/images/favicons/favicon-jetpack-cloud-production.ico",
+	"client_slug": "browser",
+	"protocol": "http",
+	"hostname": "agencies.automattic.com",
+	"i18n_default_locale_slug": "en",
+	"site_name": "Automattic For Agencies",
+	"features": {
+		"a8c-for-agencies": true
+	},
+	"enable_all_sections": false,
+	"sections": {
+		"a8c-for-agencies": true
+	},
+	"site_filter": [],
+	"theme": "jetpack-cloud",
+	"restricted_me_access": false,
+	"theme_color": "#2fb41f",
```

Comment:
I don't think it is significant, but we can have #fff as our initial theme color so it closely represents its name (Calypso White). wdyt?

Reply:
I think we can leave this as-is and update it with whatever theme color we choose later.